torchmdnet/datasets/ace.py

```diff
@@ -15,15 +15,17 @@ def __init__(
         pre_transform=None,
         pre_filter=None,
         paths=None,
+        atomic_numbers=None,
         max_gradient=None,
         subsample_molecules=1,
     ):
         assert isinstance(paths, (str, list))
 
-        arg_hash = f"{paths}{max_gradient}{subsample_molecules}"
+        arg_hash = f"{paths}{atomic_numbers}{max_gradient}{subsample_molecules}"
```

Comment:
atomic_numbers is a list, this string will be something like:
```"some_path[1, 2, 3]None1"```
Previously it had no spaces, now it does. Is this ok?
Also, atomic_numbers can be [1,2] in some instance and [2,1] in another.
While being functionally equivalent they will have different hashes. Is this what you want?
Otherwise a simple sort fixes it.

Reply:
Store the elements as a set. Now the hash will be the same for different atom order.